client-examples/BrutalmazeClient.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.Diagnostics;
+
+namespace BrutalmazeClient
+{
+	class Program
+	{
+		private static Socket ConnectSocket(string host, int port)
+		{
+			IPHostEntry ipHostInfo = Dns.GetHostEntry(host);
+			IPAddress ipAdress = ipHostInfo.AddressList[0];
+			IPEndPoint remoteEP = new IPEndPoint(ipAdress, port);
+			Socket tmp = new Socket(ipAdress.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
+			tmp.Connect(remoteEP);
```

Comment:
Somehow this couldn't connect to the server on my machine:

```cs
Unhandled Exception:
System.Net.Sockets.SocketException (0x80004005): Connection refused
  at System.Net.Sockets.Socket.Connect (System.Net.EndPoint remoteEP) [0x000d1] in <bd46d4d4f7964df
a9beea098499ab597>:0 
  at BrutalmazeClient.Program.ConnectSocket (System.String host, System.Int32 port) [0x00026] in <c
c8b4399f4104d5a95c9b85e821c6408>:0 
  at BrutalmazeClient.Program.Main (System.String[] args) [0x00000] in <cc8b4399f4104d5a95c9b85e821
c6408>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.Net.Sockets.SocketException (0x80004005): Connection refu
sed
  at System.Net.Sockets.Socket.Connect (System.Net.EndPoint remoteEP) [0x000d1] in <bd46d4d4f7964df
a9beea098499ab597>:0 
  at BrutalmazeClient.Program.ConnectSocket (System.String host, System.Int32 port) [0x00026] in <c
c8b4399f4104d5a95c9b85e821c6408>:0 
  at BrutalmazeClient.Program.Main (System.String[] args) [0x00000] in <cc8b4399f4104d5a95c9b85e821
```

Changing it to [`tmp.Connect(host, port`)](https://msdn.microsoft.com/en-us/library/4xzx2d41(v=vs.110).aspx) works though. After that L18 can be discarded I think.